### DISCUSSION TITLE
How to get NetSuite Subsidiary name after linking with codat

### DISCUSSION THREAD

**Discussion by thomas-mathers** [codatio/codat-docs#724] [Q&A] [answered]
How to get NetSuite Subsidiary name after linking with codat

**Comment by mcclowes** [comment, marked as answer]
Assuming you've [connected each subsidiary as a separate company](https://docs.codat.io/integrations/accounting/netsuite/oracle-netsuite-faq#what-is-netsuite-oneworld), you should then be able to call `GET https://api.codat.io/companies/{companyId}/data/info` which should return info about the subsidiary inc. name.
